mastering-plone/thirdparty_behaviors.rst

```diff
@@ -25,7 +25,7 @@ To use it add the name to your list of eggs in :file:`buildout.cfg`:
         ...
         collective.behavior.banner
 
-Even though :py:mod:`collective.behavior.banner` has been released on pypi we'll now will act as if this addon exists on github only - or alternatively has changes on github that have not been released yet but you that you really want. This is not to annoy you - it happens surprisingly often if you work with new versions of Plone.
+Even though :py:mod:`collective.behavior.banner` has been released on pypi we'll now act as if this addon exists on github only - or alternatively has changes on github that have not been released yet but you that you really want. This is not to annoy you - it happens surprisingly often if you work with new versions of Plone.
```

Comment:
- s/pypi we'll/PyPI, we will
- s/github only - or alternatively/GitHub, or
- s/github that have not been released yet but you that you really want./GitHub that have not yet been released, but that you really want.
- s/annoy you - it happens/annoy you. It happens
- Break all sentences onto their own line